include/ziapi/Version.hpp

```diff
@@ -8,9 +8,34 @@ namespace ziapi {
 struct Version {
     Version(int major, int minor) : major(major), minor(minor) {}
 
-    bool operator==(const Version &other) const { return this->major == other.major && this->minor == other.minor; }
+    inline bool operator==(const Version &other) const noexcept
+    {
+        return this->major == other.major && this->minor == other.minor;
+    }
 
-    bool operator!=(const Version &other) const { return this->major != other.major || this->minor != other.minor; }
+    inline bool operator!=(const Version &other) const noexcept { return !(*this == other); }
+
+    inline bool operator>(const Version &other) const noexcept
+    {
+        if (this->major > other.major)
+            return true;
+        if (this->major < other.major)
+            return false;
+        return this->minor > other.minor;
+    }
+
+    inline bool operator<(const Version &other) const noexcept
+    {
+        if (this->major < other.major)
+            return true;
+        if (this->major > other.major)
+            return false;
+        return this->minor < other.minor;
+    }
```

Comment:
```suggestion
    inline bool operator<(const Version &other) const noexcept
    {
        if (major < other.major)
            return true;
        if (major > other.major)
            return false;
        return minor < other.minor;
    }
```
I'd remove the `this` since it is not necessary in C++ and I find it more readable without it.
Anyway it is not a big deal just preferences IMO.